criu/net.c

```diff
@@ -111,15 +111,18 @@ int read_ns_sys_file(char *path, char *buf, int len)
 	}
 
 	rlen = read(fd, buf, len);
+	if (rlen == -1)
+		pr_perror("Can't read ns' %s", path);
 	close(fd);
 
 	if (rlen == len) {
+		buf[len - 1] = 0;
```

Comment:
nit: it's C so we don't have to be picky about types, but maybe use `\0` as it is more suitable here.